src/ioctls/system.rs

```diff
@@ -81,12 +114,23 @@ impl Kvm {
 
     /// Checks if a particular `Cap` is available.
     ///
+    /// Returns true if the capability is supported and false otherwise.
     /// See the documentation for `KVM_CHECK_EXTENSION`.
-    /// Returns "0 if the capability is unsupported or a positive integer otherwise.
     ///
     /// # Arguments
     ///
-    /// * `c` - KVM capability.
+    /// * `c` - KVM capability to check.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use kvm_ioctls::Kvm;
+    /// use kvm_ioctls::Cap;
```

Comment:
Why does `use Kvm` need `#` while `use Cap` doesn't?

Reply:
use Cap is firstly used in the comments here. Future references to Cap will not include this line. Same happens fro use Kvm. But I can also comment both.